### PR TITLE
Content updates for review for GA

### PR DESCRIPTION
**Pull request by KCCross**
Changes to text strings in some source files. Rewritten readme file for the repo home page as well.

### CODE REVIEW

File: src/customer-key-store/appsettings.json [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
   },
   "TestKeys": [
     { 
       "Name": "YourTestKeyName",
       "Id": "GUID",
-      "AuthorizedRoles": ["On prem AD groups that should have access to this key, if this value is present then the authorized emails will not be used"],
-      "AuthorizedEmailAddress": ["Emails of users that have access to this key. "],
-      "PublicPem" :  "The public key in PEM format.  It should not include the BEGIN and END lines",
-      "PrivatePem":  "The private key in PEM format.  It should not include the BEGIN and END lines"
+      "AuthorizedRoles": ["On premises Active Directory groups that you want to have access to this key. If you provide a value for AuthorizedRoles, then don't use AuthorizedEmailAddress."],
```
Comment (k-bren) [inline src/customer-key-store/appsettings.json:28]: >then don't use AuthorizedEmailAddress [](start = 149, length = 37)

then AuthorizedEmailAddress must be removed

File: src/customer-key-store/appsettings.json [hunk 2]
```diff
@@ -19,16 +19,16 @@
...
+      "AuthorizedEmailAddress": ["Email addresses of users that have access to this key. If you provide a value for AuthorizedEmailAddress, then don't use AuthorizedRoles."],
```
Comment (k-bren) [inline src/customer-key-store/appsettings.json:29]: >then don't use AuthorizedRoles [](start = 140, length = 30)

then AuthorizedRoles must be removed

**Review by k-bren — CHANGES_REQUESTED** [review]
:clock1:

File: src/customer-key-store/appsettings.json (hunk 1 above)
Comment (KCCross) [reply src/customer-key-store/appsettings.json:28]: "AuthorizedRoles": ["On premises Active Directory groups that you want to have access to this key. If you provide a value for AuthorizedRoles, then remove the line that starts with AuthorizedEmailAddres."],
      "AuthorizedEmailAddress": ["Email addresses of users that have access to this key. If you provide a value for AuthorizedEmailAddress, then remove the line that starts with AuthorizedRoles."],

File: src/customer-key-store/appsettings.json (hunk 2 above)
Comment (KCCross) [reply src/customer-key-store/appsettings.json]: see previous

**Review by k-bren — APPROVED** [review]
:shipit:

**Pull request merged**